lib/rules/naming/non-state-var-no-leading-underscore.js

```diff
@@ -0,0 +1,32 @@
+const BaseChecker = require('./../base-checker')
+const naming = require('./../../common/identifier-naming')
+
+const ruleId = 'non-state-var-no-leading-underscore'
+const meta = {
+  type: 'naming',
+
+  docs: {
+    description: `State variable name not start with underscore.`,
```

Comment:
Can you make this description a little more clear? Maybe `Only state variables can start with an underscore` or something like that?